docs/python/ReadMeOV.rst

```diff
@@ -7,7 +7,7 @@ OpenVINO™ Execution Provider for ONNX Runtime accelerates inference across man
  - Intel® CPUs
  - Intel® integrated GPUs
  - Intel® discrete GPUs
- - Intel® integrated VPUs
+ - Intel® integrated NPUs
```

Comment:
Let us remove from ReadME until we have approval from CCG

Reply:
Removed NPU from ReadME @sfatimar